.github/workflows/npm_build.yml

```diff
@@ -0,0 +1,21 @@
+name: NodeJS Build
+
+on:
+  push:
```

Comment:
on: push, make it consistent with other Github Actions :)

Reply:
Fixed.